src/main/java/org/apidesign/demo/matrixultimate/svm/SVMScientificLibraryJNI.java

```diff
@@ -10,129 +10,118 @@
 import org.graalvm.nativeimage.c.type.CTypeConversion;
 import org.graalvm.word.Pointer;
 
-/** Native image implementation of JNI entry point methods. All the
- * {@link CEntryPoint} methods are on the "boundary" from the native side.
- * The code in here can interface with C libraries without any overhead.
+/**
```

Comment:
The changes in this file look like reformatting only. Can we please exclude reformatting related changes from the PR?

Reply:
I fixed that. Sorry (too many opinions on Java format rules out there - yours are not the same as the other projects I work on apparently).